RELEASES.md

```diff
@@ -1,3 +1,117 @@
+Version 1.84.0 (2025-01-09)
+==========================
+
+<a id="
+Language"></a>
+
+Language
+--------
+- [Allow `#[deny]` inside `#[forbid]` as a no-op](https://github.com/rust-lang/rust/pull/121560/)
+- [Show a warning when `-Ctarget-feature` is used to toggle features that can lead to unsoundness due to ABI mismatches](https://github.com/rust-lang/rust/pull/129884)
+- [Use the next-generation trait solver in coherence](https://github.com/rust-lang/rust/pull/130654)
```

Comment:
Can we also duplicate this into compat notes ("Fix some soundness bugs in coherence checking") or so?